frontend/src/pages/projects/screens/projects/useProjectTableRowItems.ts

```diff
@@ -69,11 +51,11 @@ const useProjectTableRowItems = (
     },
     {
       title: 'Edit permissions',
-      isAriaDisabled: !allowCreate || !allowCreateLoaded,
+      isAriaDisabled: !allowUpdate || !allowUpdateLoaded,
       onClick: () => {
         navigate(`/projects/${project.metadata.name}?section=permissions`);
       },
-      ...noPermissionToolTip(allowCreate, allowCreateLoaded),
+      ...noPermissionToolTip(allowUpdate, allowUpdateLoaded),
```

Comment:
During our conversation, this actually is accurate to being needing create against Role Bindings -- because that's what this tab does...